documentation/ipi-install/modules/ipi-install-config-yaml.adoc

```diff
@@ -21,7 +21,7 @@ platform:
   baremetal:
     apiVIP: <api-ip>
     ingressVIP: <wildcard-ip>
-    dnsVIP: <dns-ip>
+    dnsVIP: <dns-ip>  # Deprecated in 4.5 https://github.com/openshift/installer/pull/3304
```

Comment:
Can we create a new install-config-yaml-4.5.adoc and just remove it fromt here entirely? I know its more yaml's to maintain but keeps it cleaner and if user copy pastes they aren't copying pasting potentially an old value.

Reply:
According to what we spoke in the past, I've created a -4.4 to leave it there, so that 4.5 or higher doesn't have them (so that it's easier to find deprecated content in the future)